docs/libraries/messaging.md

```
@@ -1,775 +1,825 @@
+
+### Local Development with Docker Compose
+
+Start RabbitMQ and other infrastructure:
+
+```bash
+docker compose -f deploy/compose/docker-compose.dev.yml up -d
+```
+
+Access RabbitMQ Management UI:
+
+- **URL**: http://localhost:15672
```

markdownlint-cli2 (0.20.0)

Bare URL used

(MD034, no-bare-urls)